recipes/pantax/meta.yaml

```
@@ -0,0 +1,57 @@
+requirements:
+  build:
+    - {{ compiler('rust') }}
+  host:
+    - python ==3.10
+    - pip
+  run:
+    - python =3.10
+    - samtools =1.19.2
+    - bcftools =1.19
+    - htslib =1.19.1
+    - pggb =0.5.4 
```

yamllint

[error] trailing spaces

(trailing-spaces)